src/coreclr/gc/gc.h

```diff
@@ -124,7 +124,10 @@ enum gc_generation_num
     ephemeral_generation_count = max_generation,
 
     // number of all generations
-    total_generation_count = poh_generation + 1
+    total_generation_count = poh_generation + 1,
+
+    // number of uoh generations
+    uoh_generation_count = total_generation_count - ephemeral_generation_count
 };
```

Comment:
this should be 2, not 3, ie,

`total_generation_count - uoh_start_generation`

Reply:
wow, I must have read that as "loh_generation_count" which of course is different.  fixed.